src/components/composer/composer/abstract_composer_store.ts

```diff
@@ -609,6 +601,37 @@ export abstract class AbstractComposerStore extends SpreadsheetStore {
     return (token.parenthesesCode || "").startsWith(previousSymbolAtCursor.parenthesesCode || "");
   }
 
+  private computeParenthesisRelatedToCursor() {
+    if (
+      this.tokenAtCursor &&
+      ["LEFT_PAREN", "RIGHT_PAREN"].includes(this.tokenAtCursor.type) &&
```

Comment:
On Gsheet the parenthesis are highligted if the cursor is AFTER the opening parenthesis, or BEFORe the closing parenthesis

Reply:
I don't see this behavior on gsheet, on excel perhaps ?

Reply:
MMh yeah it seems i'm hallucinating now ....

Reply:
perhaps, but tell me you see walls that go into the ground, castles that move and viking boats coming out of the water... I would believe you